drivers/serial/uart_stm32.c

```diff
@@ -395,6 +395,46 @@ static void uart_stm32_poll_out(struct device *dev,
 	LL_USART_TransmitData8(UartInstance, (u8_t)c);
 }
 
+static int uart_stm32_err_check(struct device *dev)
+{
+	USART_TypeDef *UartInstance = UART_STRUCT(dev);
+	int err = 0;
+
+	if (LL_USART_IsActiveFlag_ORE(UartInstance)) {
+		err |= UART_ERROR_OVERRUN;
```

Comment:
Should be ok but, I'll wait feedback on https://github.com/zephyrproject-rtos/zephyr/issues/13256 before approving this.

Reply:
#13256 has been closed, rebased the repo